src/Table/Table.tsx

```diff
@@ -0,0 +1,985 @@
+                                        :
+                                        ((row.subRows.length > 0 || row.rowContentDetail) && props.rowsAreCollapsable) &&
+                                        <td>
+                                            <div
+                                                className={"icon-holder" + (row.expanded ? " active" : "")}
+                                                onClick={(e: React.MouseEvent<HTMLDivElement, MouseEvent>) => { props.onRowExpanded(e, row); }}
+                                            >
+                                                {row.expanded ? angleDown : angleRightIcon}
+                                            </div>
+                                        </td>
+                                    }
+                                    {row.cells.map((cell: Cell, cellIndex: number) => {
+                                        return <td key={`cell-${cellIndex}`}>
+                                            {cell.value}
+                                        </td>;
+                                    })}
+                                    {props.useShowActionColumn &&
+                                        <td>
+                                            <ActionColumn
+                                                actionLinks={props.actionLinks}
+                                                primaryActionButton={props.primaryActionButton}
+                                                selectedRow={row}
+                                                tableRef={tableRef}
+                                                onActionDropped={(event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+                                                    props.onActionDropped(event, row, null);
+                                                }}
+                                            />
+                                        </td>
+                                    }
+                                </tr>
+
+                                {row.subRows.map((subRow: TableRow) => {
+                                    const checkSubRowRandomIds: string = randomId("sub-chk-");
+                                    return (
+                                        <React.Fragment key={`sub-row-${subRow.rowIndex}`}>
+                                            <tr
+                                                className={"sub-row" + (subRow.expanded ? " expanded" : "")}
+                                                style={{ display: row.expanded ? "table-row" : "none" }}
+                                            >
+                                                {props.useRowSelection ?
+                                                    <td className="row-selections-column">
+                                                        <div className="custom-control custom-checkbox" style={{ marginLeft: "20px" }}>
```

Comment:
Why is this hardcoded in JS? `style={{ marginLeft: "20px" }}` Just add it to the stylesheet. This will make it difficult for users to re=style the component. The only solution for them might be using `!important`

I found a bunch of these around this file. Just find them and remove them from JS and add them to CSS